R/jira.R

```diff
@@ -8,15 +8,16 @@
 
 #' Parse Jira issue and comments
 #'
+#' Note the comments element will be empty if the downloaded json only contain issues.
+#' the jira json naming convention is as follows: "(ProjectKey)_issues_(UNIXTIMElowerbound)_(UNITITMEupperbound).json"
+#' or "(ProjectKey)_issue_comments_(UNIXTIMElowerbound)_(UNIXTIMEupperbound).json".
+#' For example: "KAIAULU_issues_1231234_2312413.json". Notice how the ProjectKey portion is in all caps.
+#'
 #' @param json_path is a folder path containing a set of jira_issues as json files. The general folder structure is as follows:
```

Comment:
Overall folder structure should be moved to the description. This should also link to the downloader function.

Reply:
Ok (I added a link to Sean's downloader function and put the general folder structure text in the description; before, it was in the param section).